### DISCUSSION TITLE
Readme as landing page for organisations

### DISCUSSION THREAD

**Discussion by joostschouppe** [community/community#23711] [New to GitHub]
<p>It is possible to set a readme as a landing page for organisations? I followed <a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-profile/managing-your-profile-readme" rel="noopener nofollow ugc">the official documentation</a>) but it seems like this only works for individual accounts</p>
<p>(note: I also commented the same question at <a href="https://github.community/t/profile-readme-md-section-for-organisations/135254/2">this solved question</a> but that’s probably not the best way to ask this)</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Yes, the feature that allows for a README in a specific repository to show up on the profile page for a user is only implemented for user accounts at this time, not organization accounts. We don’t have any information as to when or whether this feature will be implemented for organizations as well. I’d recommend keeping an eye on <a href="https://github.blog/changelog/">the changelog</a> and the <a href="https://github.com/github/roadmap">official GitHub public roadmap</a> for updates.</p>
<p>I hope that helps!</p>
